src/components/MenuBar/ButtonBar.js

```diff
@@ -69,7 +69,7 @@ function ButtonBar({ isMobile }) {
 				<PoppinLink to={link.to} key={`nav-${index}`}>
 					{link.name}
 				</PoppinLink>)}
-			<BorderLink href='https://forms.gle/7uokDycPQfU9B5oj8' target='_blank'>
+			<BorderLink href={'https://forms.gle/7uokDycPQfU9B5oj8'} target='_blank'>
```

Comment:
This is a remnant of the previous branch you based this branch off of, we should remove the change from here.